### ISSUE TITLE
National Map v2018-04-16 pre-release test - Elevation counter

### ISSUE THREAD

**Issue by EhsanCanberra** [TerriaJS/nationalmap#731]
**Observation**

When testing the combination of available Map Views and Base Maps - the elevation counter briefly disappeared - it self reset later. Not sure why.

![slide11](https://user-images.githubusercontent.com/22827786/39111483-d06da87e-46c4-11e8-8df1-ac81d45b7452.JPG)

**Comment by AnaBelgun** [comment]
I think elevation counter usually shows values (works) when the mouse pointer is on the map; it disappears/shows no values when the mouse is on the Map View or other menu view. @kring please confirm, but I don't see any errors with this behaviour.

**Comment by kring** [comment]
The only way I found to get the view in Ehsan's screenshot (where the latitude and longitude are shown by elevation is blank) is by switching to 3D Smooth or 2D. However, in Ehsan's screenshot, 3D Terrain is clearly selected. So I found one small bug that I _think_ explains the screenshot: toggling between 3D Terrain and 3D Smooth does _not_ automatically restart the Cesium render loop. So, if you switch to 3D Smooth, keep the mouse still inside the Map panel for a few sounds, and then click on the 3D Terrain button, you'll be able to get a screenshot like Ehsan's. As soon as you move the mouse back over the map, rendering restarts, the globe switches to 3D Terrain, and the elevation display reappears (or the latitude and longitude disappear, too, if the cursor isn't over the globe).

I wrote TerriaJS/terriajs#2952 to cover the failure to repaint automatically when switching 3D modes.
